src/components/templates/FireBarrel/FireBarrel.tsx

```diff
@@ -46,29 +49,69 @@ const FireBarrel: React.FC = () => {
     }
   }, [partygoers, venue]);
 
+  const { user } = useUser();
+
+  const { room } = useVideoState({ userUid: user?.uid, roomName: venue?.name });
+
+  const chairsArray = Array.from(Array(chairs));
+
+  const [videoError, setVideoError] = useState<string>("");
+
   return useMemo(
     () => (
       <S.Wrapper>
         <S.Barrel src={ConvertToEmbeddableUrl(venue?.iframeUrl)} />
 
-        {Array.from(Array(chairs)).map((_, index) => {
+        {chairsArray.map((_, index) => {
           const partyPerson = currentPartygoers[index] ?? null;
+
+          const isMe = partyPerson?.id === user?.uid;
+
           if (!partyPerson) {
             return <S.Chair key={index} isEmpty />;
           }
 
-          return (
-            <FireSeat
-              key={index}
-              person={partyPerson}
-              roomName={venue?.name}
-              chairNumber={index}
-            />
-          );
+          if (!!room && isMe) {
+            return (
+              <LocalParticipant
+                participant={room?.localParticipant}
+                profileData={profileData}
+                profileDataId={room?.localParticipant.identity}
+                showIcon={false}
+              />
+            );
+          }
+
+          if (!!room && !isMe) {
+            return (
+              <RemoteParticipant
+                participant={room?.TODO}
+                profileData={profileData}
+                profileDataId={room?.localParticipant.identity}
+                showIcon={false}
+              />
+            );
+          }
+
+          // return (
+          //   <S.Chair isEmpty={false}>
+          //     // TODO make the styled component stuff work again
+          //   </S.Chair>
+          // );
```

Comment:
Need to reinstate the styled-components stuff here, wasn't sure how it worked and din't want to waste too much time to get this sub-PR up.